/**
 * throw result of `toError`
 *
 * @example
 * ```ts
 * const result: never = pipe(
 *   0,
 *
 *   throwError((input) => Error(`input is ${input}`)),
 *
 *   (input: never) => {
 *     // unreachable point
 *   },
 * );
 * ```
 */
var throwError = function (toError) {
    return function (input) {
        throw toError(input);
    };
};
export default throwError;
//# sourceMappingURL=throwError.js.map